Doc/library/importlib.rst

```diff
@@ -1385,6 +1385,10 @@ an :term:`importer`.
    ``/foo/bar/__pycache__/baz.cpython-32.opt-2.pyc``. The string representation
    of *optimization* can only be alphanumeric, else :exc:`ValueError` is raised.
 
+   The *noopt* parameter is used to specify if compiler optimization are
+   disabled. If it is true, *optimization* is used and ``.noopt`` suffix is
```

Comment:
"If it is true, *optimization* is used" . From the below code it seems optimization code path is taken when noopt false at else clause https://github.com/python/cpython/pull/13600/files#diff-42218ca77825a5fef3c3d62a9f6b1304R327.

Reply:
Update the docstring for `cache_from_source` too about `noopt` flag?

Reply:
You should read "optimization is ignored" :-) Fixed.